### DISCUSSION TITLE
How to implement custom path? I'm in a hurry, Thank you for your reply!!!!!!

### DISCUSSION THREAD

**Discussion by parlay96** [vercel/next.js#22654] [Help]
**Describe the feature you'd like to request**

How to implement custom path? For example: {path: '/ home.abc ', component: import('../viewPage/home') } 。 Path is something I can control, so how can next do it?

**Describe the solution you'd like**

How to implement custom path? For example: {path: '/ home.abc ', component: import('../viewPage/home') } 。 Path is something I can control, so how can next do it?

**Describe alternatives you've considered**

How to implement custom path? For example: {path: '/ home.abc ', component: import('../viewPage/home') } 。 Path is something I can control, so how can next do it?

How to implement custom path? For example: {path: '/ home.abc ', component: import('../viewPage/home') } 。 Path is something I can control, so how can next do it?

**Comment by andrewmclagan** [comment]
This is not an issue. This is a discussion: https://github.com/vercel/next.js/discussions

Open source software is built by people in their own time.
